packages/components/file-input/src/FileInput.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Button } from '@gpn-prototypes/vega-button';
+import { IconAttach } from '@gpn-prototypes/vega-icons';
+import { action } from '@storybook/addon-actions';
+import { text, withKnobs } from '@storybook/addon-knobs';
+import { storiesOf } from '@storybook/react';
+
+import { FileInput, FileInputProps } from './FileInput';
+
+const defaultKnobs = (): FileInputProps => ({
+  id: text('id', 'id'),
+});
+
+storiesOf('ui/FileInput', module)
+  .addDecorator(withKnobs)
+  .addParameters({ metadata: { author: 'CSSSR', status: 'Approved' } })
+  .add('FileInput', () => {
```

Comment:
продублируется имя сторисы
```suggestion
  .add('по умолчанию', () => {
```